frontend/ui/widget/touchmenu.lua

```diff
@@ -594,16 +594,25 @@ function TouchMenu:updateItems()
     table.insert(self.item_group, self.footer_top_margin)
     table.insert(self.item_group, self.footer)
     -- @translators %1 is the current page. %2 is the total number of pages. In some languages a good translation might need to reverse this order, for instance: "Total %2, page %1".
-    self.page_info_text.text = util.template(_("Page %1 of %2"), self.page, self.page_num)
+    if self.page_num > 1 then
+      self.page_info_text.text = util.template(_("Page %1 of %2"), self.page, self.page_num)
```

Comment:
To be future-proof (I was seeing blank because I had #5503 in), can you use setText() like you did in the other?
`self.page_info_text:setText(util.template(_("Page %1 of %2"), self.page, self.page_num))`

Reply:
Done, that line of code wasn't mine however :)